src/datasource/Boku no Hero.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I think this file caused an error

Reply:
Ok thanks, I will try to solve this problem

Reply:
I was deleted the file from this PR. But the problem wasn't solved. And I got an error below

"Detective-Conan is not in the right pattern. Please use kebab-case.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! animoji@0.1.0 validate:file-name: `node src/datasource/validate.js file-name`
npm ERR! Exit status 1"

Reply:
i think ur codebase is not the same as in the upstream, can you rename the Detective-Conan to detective-conan?

Reply:
I give up :(. I need learn more about Git. But thanks :)

Reply:
that's okay, just rename the file Detective-Conan to detective-conan.

Reply:
I was rename all the namefile using kebab-case but the problem is not resolved

Reply:
have u rename detective-conan and commit the changes?

Reply:
Hi there, I think there's a little mistake I made. I haven't renamed the `Detective-Conan` to `detective-conan`. You can try to rename the file and try to push a commit again.

Reply:
It's solved :). I was rename to detective-conan but the problem is not resloved. But now I rename to the japanese name (meitantei-konan) and solved.

Reply:
hi, can you use detective-conan.json instead of meitantei-konan.json ?

Reply:
Nice solving! Thanks for contributing.

Reply:
Done